pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -40,7 +40,7 @@ const (
 
 var (
 	zeroDuration = metav1.Duration{}
-	// Refer to [Node Allocatable](https://git.k8s.io/community/contributors/design-proposals/node-allocatable.md) doc for more information.
+	// Refer to [Node Allocatable](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md) doc for more information.
```

Comment:
This is still pointing to a design doc. Do we have a corresponding doc in kubernetes.io?

Reply:
because I think this will not be used to generate user documentation. Link to design proposals for developer is more appropriate.